aces/Loris
# [DQT] Performance fix for load query and fix vars exceed on save

## Brief summary of changes

Found by @pierre-p-s when lots of fields are saved. Performance part was found once we got passed that and attempted to load the saved query.

#### Testing instructions (if applicable)

1. Visit DQT
2. Add many fields
3. Save and see if everything is okay.
4. Attempt to load query after browser refresh.

#### Link(s) to related issue(s)

* Resolves #  (Reference the issue this fixes, if any.)
